packages/shared/src/components/PlusUser.tsx

```diff
@@ -16,7 +16,7 @@ export const PlusUser = ({
 }: Props): ReactElement => {
   return (
     <Typography
-      className={classNames('flex text-accent-bacon-default', className)}
+      className={classNames('text-action-plus-default flex', className)}
```

Comment:
We actually have `TypographyColor.Plus`

Reply:
I don't see it 🙈 
Do you have it in your branch?
https://github.com/dailydotdev/apps/blob/b1714c4f80cc68cb224daa0d8d6ca4665df4d62d/packages/shared/src/components/typography/Typography.tsx#L40-L48

Reply:
Yeah, found it there
https://github.com/dailydotdev/apps/blob/3eba23b2b80028cec4809f2805f8f1f5c7878e42/packages/shared/src/components/typography/Typography.tsx#L48

Reply:
Oh right, my bad thought I pushed it on the other one already.
Maybe base it on my one so long?

Reply:
Or take that change in here already :)

Reply:
I'll rebase it over yours, and change merge target to your branch